sqsdomain/pools.go

```diff
@@ -60,9 +69,10 @@ type SQSPool struct {
 }
 
 type PoolWrapper struct {
-	ChainModel poolmanagertypes.PoolI `json:"underlying_pool"`
-	SQSModel   SQSPool                `json:"sqs_model"`
-	TickModel  *TickModel             `json:"tick_model,omitempty"`
+	ChainModel  poolmanagertypes.PoolI `json:"underlying_pool"`
+	SQSModel    SQSPool                `json:"sqs_model"`
+	TickModel   *TickModel             `json:"tick_model,omitempty"`
+	CWPoolModel *CWPoolModel           `json:"cw_pool_model,omitempty"`
```

Comment:
Can we instead define `*CWPoolModel` on `SQSModel`?

That way, we can avoid defining concrete CosmWasm getter and setter APIs on the generalized `PoolI` interface, helping to keep it lean.

Pet this suggestion, we would be able to use `GetSQSPoolModel().CWPoolModel`

Reply:
sounds good! will do that